.changeset/techdocs-stale-breakfast-king.md

```diff
@@ -0,0 +1,19 @@
+---
+'@backstage/techdocs-common': patch
```

Comment:
Is this an opt-in or is it on by default? I would assume it is a breaking change if you require new IAM permissions.

Reply:
Good call-out.  The intention was backward compatibility (hence the patch), but there may be a couple more spots to ensure that.

Reply:
Updates to catch/log when listing [here](https://github.com/backstage/backstage/pull/6488/commits/c2d6c8054db8de6fe33847b9869307cadf22b54f), and we otherwise catch/log on the actual delete operations as well.